src/main/java/com/fa/training/service/FootballPlayerServiceImpl.java

```diff
@@ -20,17 +21,60 @@ public FootballPlayerServiceImpl(FootballPlayerDaoImpl footballPlayerDaoImpl) {
      * Add a football player to database
      * @param footballPlayerDto
      */
-    public void create(FootballPlayerDto footballPlayerDto) {
-        footballPlayerDaoImpl.create(new FootballPlayer(footballPlayerDto));
+    public int create(FootballPlayerDto footballPlayerDto) {
+
+        int id = (int) footballPlayerDaoImpl.create(new FootballPlayer(footballPlayerDto));
```

Comment:
I think this line should be separated into two lines.
One is to create new FootballPlayer.
Second is footballPlayerDaoImpl.create(footballPlayer). 
Don't write code in one line.

Reply:
Thank you, I will fix this line.